src/cpp/ext/proto_server_reflection.h

```diff
@@ -30,12 +30,26 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  *
  */
+
+/*
+  - If the generated header `grpc++/ext/reflection.grpc.pb.h` needs to be
+    installed, target `grpc++_reflection` in `build.yaml` should use the
+    filegroup `grpc++_reflection_proto`, and GRPC_NO_GENERATED_CODE should not
+    be defined.
+  - If the server reflection library needs to generate `reflection.grpc.pb.h`
+    from `reflection.proto` at the time of making, the generated header
```

Comment:
by "at the time of making" do you mean "at compile time"? It may be confusing later one when we switch to bazel to keep referring to making to mean "invoke make to compile".